en/ros2/px4_ros2_navigation_interface.md

```diff
@@ -0,0 +1,247 @@
+# PX4 ROS 2 Interface Library: Navigation
+
+:::warning Experimental
+At time of writing parts of the PX4 ROS 2 Interface Library is experimental, and hence subject to change.
+:::
+
+The PX4 ROS 2 Interface Library for navigation enables developers to send their position estimates to PX4 directly using ROS 2.
+The interface provides a layer of abstraction from PX4 and the uORB messaging framework, and introduces a few sanity checks on the requested state estimation updates sent via the interface.
+These estimates are then fused into the EKF as internal PX4 estimates would.
```

Comment:
I'd probably expand on the use cases here that you envisage - presumably providing position via computer vision, or allowing a mission computer that has a GPS to provide that. 

Also do you need to do anything in PX4 - i.e. turn off other sources?

Reply:
I guess I'm also thinking about the "parent" library, which is mostly about managing offboard control modes such that they have an identity and can safely coexist. The question being, how this related to that or whether it is fully independent.